src/transform_output/transform_output.cpp

```diff
@@ -68,149 +68,199 @@ void transform_output(const char *ip_file_name, const char *op_file_name) {
 	 * setting the column names for our output file.
 	 * setw() function is used to give indentation in the output file.
 	 */
-	outputfile <<setw(7)<<"Time"<<setw(20)<<"Value"<<setw(14) \
-			   <<"Port"<<setw(22)<<"Component"<<endl;
-	outputfile <<"-------------------------------------------"
-				 "---------------------------"<<endl;
+	//outputfile <<setw(7)<<"Time"<<setw(20)<<"Value"<<setw(14) \
+	//		   <<"Port"<<setw(22)<<"Component"<<endl;
+	outputfile << setw(7) << "Time" << setw(20) << "Value" << setw(22) \

```

Comment:
no need of slash as the end for continuation

Reply:
modified as requested

Reply:
\ at the end is not required it will work without \

Reply:
thank you for the note. modified as per requested.